From 3f831d30d87658d362e9b4f25fa569937fa1c23f Mon Sep 17 00:00:00 2001
From: Gustavo Luiz Duarte <gustavold@linux.ibm.com>
Date: Tue, 31 Mar 2020 23:11:44 -0300
Subject: [PATCH] supermin: Fix IBM Virtual SCSI driver name

The driver was renamed from ibmvscsic to ibmvscsi on kernel 3.7.
See https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=9965c2f19be470c452357ae4f6304467cdeada55

The old name is kept so supermin works with recent kernels as well as kernel
version older than 3.7.
---
 src/format_ext2_initrd.ml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/format_ext2_initrd.ml b/src/format_ext2_initrd.ml
index b67bb55..38977e6 100644
--- a/src/format_ext2_initrd.ml
+++ b/src/format_ext2_initrd.ml
@@ -44,6 +44,7 @@ let kmods = [
   "crc*.ko*";
   "libcrc*.ko*";
   "ibmvscsic.ko*";
+  "ibmvscsi.ko*";
   "libnvdimm.ko*";
   "nd_pmem.ko*";
   "nd_btt.ko*";
-- 
2.24.1

